pkg/email/templates/system_intake_submit_business_case_reviewer.gohtml

```diff
@@ -0,0 +1,13 @@
+{{template "easi_header.gohtml"}}
+
+<p>{{.RequesterName}} has {{if .IsResubmitted}}made changes to their{{else}}submitted a{{end}} {{if .IsDraft}}draft{{else}}final{{end}} Business Case for {{.RequestName}}. Use the link below to review the {{if .IsResubmitted}}changes{{else}}Business Case{{end}} in EASi. The lead for this request of another member of the Governance team should respond within two business days with any feedback about the Business Case or to move the request to the next step in the Governance Review process.</p>
+
```

Comment:
the t in Governance Team should be capitalized https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=2202-14572&mode=design&t=1W2JkzMlxIJZZAaQ-0